### PR TITLE
Allow dry flag to be used with rename

### LINKED ISSUE #2: Add `--dry` operation
**Amazing** little project you have! Very thankful for it!

Would be nice if there was a `--dry` option to see the changes it would make instead of fully committing them (And then having to use `git diff` to see the changes)

### PR DESCRIPTION
**Pull request by gopar**
Closes #2

### CODE REVIEW

File: tests/test_all.py [hunk 1]
```diff
@@ -122,6 +135,23 @@ def test_rename_module():
...
+    with runner.isolated_filesystem():
+        veg, fruit = make_veg_fruit_pyfiles()
+
+        with CaptureOutput() as output:
+            run(runner, "rename", "--dry", fruit, "fruity")
+            assert output.getvalue()
+
+        with CaptureOutput() as output:
+            run(runner, "rename", "--dry", f"{fruit}::cherries", "berries")
+            assert output.getvalue()
```
Comment (gopar) [inline tests/test_all.py:149]: Not exactly happy with this very bare bones assertion but should suffice

File: ropecli/__init__.py [hunk 2]
```diff
@@ -231,7 +232,10 @@ def very_sure(_):
     renamer = rope.refactor.rename.Rename(project, resource, offset)
     changes = renamer.get_changes(new_name, docs=True, unsure=very_sure)
 
-    project.do(changes)
+    if dry:
+        print(changes.get_description())
```
Comment (gopar) [inline ropecli/__init__.py:236]: Also not sure I like just putting a `print` statement there but I don't know much on how console applications take care of things like this

**Comment by aevri** [comment]
Thanks for contributing! I'll aim to review this week.

File: ropecli/__init__.py (hunk 2 above)
Comment (aevri) [reply ropecli/__init__.py:236]: Seems good to me, thanks!

File: tests/test_all.py (hunk 1 above)
Comment (aevri) [reply tests/test_all.py:149]: Have added a check for `Renaming`, seems slightly better now

**Comment by aevri** [comment]
I thought I'd be able to push my changes to this branch due to the "maintainers are allowed to edit this pull request", unfortunately not! I'll land and patch on top.

**Comment by aevri** [comment]
Thanks! ⭐

**Pull request merged**